app/controllers/landing_page_controller.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Controller for the landing page
+class LandingPageController < ApplicationController
+  def index
+    @featured_items = Settings.featured_items
+    @hours = {
```

Comment:
Should we drive this from a config? Then we could iterate over each configured hours location and have fewer details in the controller and template.

Should we add Special Collections? https://library-hours.stanford.edu/libraries/spc

Reply:
I'm also fine merging this and coming back to it as a separate issue.

Reply:
Pushing it to config was one of the things I was alluding to debating earlier. Seems like a good idea, I went ahead and took a shot at it.

Special Collections is another thing I need to check on. I assume, in this code (well config now), that 'Green Library, 2nd Floor' *is* Special Collections because that was the only location that matched the example hours. I'll ask about it in the issue.